commands/miner.go

```diff
@@ -376,17 +377,7 @@ var minerOwnerCmd = &cmds.Command{
 		if err != nil {
 			return err
 		}
-
-		bytes, err := GetPorcelainAPI(env).MessageQuery(
```

Comment:
these calls and the below were moved to `mine_util.go` since the `mining` and `miner` use them